index.js

```diff
@@ -23,6 +30,11 @@ module.exports = {
 
     this.app = app;
 
+    if (this._shouldIncludeFiles()) {
+      this.import('vendor/shims/-jquery.js', { prepend: true });
+      this.import('vendor/ecpo-jquery/dist/jquery.js', { prepend: true });
```

Comment:
ensure shims goes right after our private `jquery` instance

Reply:
Should we put a comment here indicating that this ordering is important for `shims/-jquery` to work? The comment in `shims/-jquery` is really helpful, and I think the assumption we're making there about ordering will be more secure if there is also a warning here, where that ordering is defined.